src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -309,9 +257,9 @@ private void maskBoolean(MaskingState maskingState, KeyMaskingConfig keyMaskingC
     private static void stepOverValue(MaskingState maskingState) {
         switch (maskingState.byteAtCurrentIndex()) {
             case '"' -> stepOverStringValue(maskingState);
-            case '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '-' -> stepOverNumericValue(maskingState);
-            case 't', 'n' -> maskingState.setCurrentIndex(maskingState.currentIndex() + 4); // true or null
-            case 'f' -> maskingState.setCurrentIndex(maskingState.currentIndex() + 5); // false
+            case '-', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9' -> stepOverNumericValue(maskingState);
```

Comment:
Keeping OCD [in check](https://github.com/Breus/json-masker/pull/96/files#diff-e8659d668c0ffdcb0d5d6fa250b0d23ad7e2947f9d5eb2d8575b4e2b04065179R74) 😅